homeassistant/components/shelly/sensor.py

```diff
@@ -318,10 +319,78 @@ class RestSensorDescription(RestEntityDescription, SensorEntityDescription):
         sub_key="apower",
         name="Power",
         native_unit_of_measurement=UnitOfPower.WATT,
-        value=lambda status, _: None if status is None else round(float(status), 1),
         device_class=SensorDeviceClass.POWER,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    "a_act_power": RpcSensorDescription(
+        key="em",
+        sub_key="a_act_power",
+        name="Phase A Active Power",
```

Comment:
One more thing 😊 We should probably use the new entity naming style.
```suggestion
        name="Phase A active power",
```

Reply:
Yes, we should use the new style, fixed for this PR, we should make another PR to fix existing sensors